front_end/html/course_admin.html

```diff
@@ -100,10 +100,10 @@ <h2>{{ course_basics["title"] }}</h2>
 
     <div>
         <a class="button is-small is-rounded is-primary" href="/edit_assignment/{{ course_basics['id'] }}/">New assignment</a>
-        <p class="buttons is-pulled-right">
-            <a class="button is-small is-rounded is-light" href="/export_course/{{ course_basics['id'] }}">Export course</a>
-            <a class="button is-small is-rounded is-light" href="/export_submissions/{{ course_basics['id'] }}">Export submissions</a>   
-        </p>
+        <!--<p class="buttons is-pulled-right">
```

Comment:
If you think we might want this code in the future, it's fine to leave it here commented. If not, please remove it. If you do keep it, please leave a comment on what it does so that we will know in the future what it does.

Reply:
These are the export buttons you asked me to comment out so you can add them back once you fix that functionality.